src/Microsoft.DotNet.Darc/src/DarcLib/AzureDevOpsClient.cs

```diff
@@ -583,135 +675,56 @@ public async Task UpdatePullRequestCommentAsync(string pullRequestUrl, string co
             return (threadId, commentIdValue);
         }
 
-        public async Task CommentOnPullRequestAsync(string pullRequestUrl, string message)
+        /// <summary>
+        /// Parse a repository url into its component parts
+        /// </summary>
+        /// <param name="repoUri">Repository url to parse</param>
+        /// <returns>Tuple of account, project, repo</returns>
+        public static (string accountName, string projectName, string repoName) ParseRepoUri(string repoUri)
```

Comment:
>public [](start = 8, length = 6)

Is this public for testing? In general, public things that return tuples are... worrisome, especially when it's just (string,string,string). Any renaming or reordering of these returned types will lead to havoc later.  A struct/class with named members is less prone to trouble and easier to document.

Reply:
Yeah public just for testing.